### ISSUE TITLE
Different path for http2

### ISSUE THREAD

**Issue by wmertens** [h2o/h2o#392]
If you could send a different index.html for http2, you'd be able to have http1 optimized html with inclining etc and http2 optimized html with server push...

**Comment by kazuho** [comment]
Sounds interesting.

There would be other reasons to switch files being served depending on certain aspects of the requests, and for such purpose I think the right solution would be to provide a way to script rewriting the requests which is under work in #387.

**Comment by wmertens** [comment]
I agree in principle, but I wonder if this will be a very common use case,
to be better handled in C than in Ruby. The future will tell.

An example script would be nice…

On Sat, Jul 11, 2015, 06:33 Kazuho Oku <notifications@github.com> wrote:

> Sounds interesting.
>
> There would be other reasons to switch files being served depending on
> certain aspects of the requests, and for such purpose I think the right
> solution would be to provide a way to script rewriting the requests which
> is under work in #387 <https://github.com/h2o/h2o/pull/387>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/h2o/h2o/issues/392#issuecomment-120572138>.
>
-- 

Wout.
(typed on mobile, excuse terseness)
